components/Newsfeed/Newsfeed.tsx

```diff
@@ -122,11 +131,67 @@ export default function Newsfeed() {
               {t("bill_updates")}
             </label>
           </BillCol>
+          <Buttons profile={profile} />
         </Row>
       </>
     )
   }
 
+  function Buttons({ profile }: { profile: Profile }) {
+    const {
+      public: isPublic,
+      notificationFrequency: notificationFrequency
+    }: Profile = profile
+
+    const [settingsModal, setSettingsModal] = useState<"show" | null>(null)
```

Comment:
Not needed for this PR, just wanted to make a note while it was on my mind:

Should we DRY up some of this settings modal stuff? I think we have the `profile` fetched on most pages anyway via the service provider - could we move the `useProfile` hook into the SettingsModal itself to cut down on the prop passing we've needed for these?